src/parser/mod.rs

```diff
@@ -12364,6 +12385,43 @@ mod tests {
         );
     }
 
+    #[test]
```

Comment:
hmm I think this would probably not be the ideal place for the test to live - if the feature isn't bound to any dialect, or if it uses a dialect method to guard the feature, we can add the tests to common e.g. https://github.com/sqlparser-rs/sqlparser-rs/blob/main/tests/sqlparser_common.rs#L4405